crates/wg_packet/src/message.rs

```diff
@@ -1,69 +1,91 @@
-use crate::packet::Fragment;
-use wg_network::topology::ServerType;
+use serde::de::DeserializeOwned;
+use serde::{Deserialize, Serialize};
 use wg_network::{NodeId, SourceRoutingHeader};
 
 #[derive(Debug, Clone)]
-pub struct Message {
-    pub message_data: MessageData,
+pub struct Message<M: DroneSend> {
```

Comment:
I think that DroneSend is confusing. Because we are using it also to deserialize and not only to "send". I'd prefer `Serializable` as before, just because it means that it contains both serialize and deserialize (even if now they are `stringify` and `from_string`). Or if you can come up with something else more generic it should be fine anyway.

Reply:
I was exploring different namings, I was trying to make it so that that trait just implies "this can be sent with drones".